frontend/src/homePage/LandingPage.js

```diff
@@ -15,6 +15,7 @@ class LandingPage extends React.Component {
         this.clickHandling = this.props.folderClick;
       }
 
+      
```

Comment:
No extra newlines at the bottom of methods.

Reply:
done